src/runtime/virtcontainers/container.go

```diff
@@ -656,6 +656,8 @@ func (c *Container) createBlockDevices(ctx context.Context) error {
 						continue
 					}
 					c.mounts[i].FSGroupChangePolicy = volume.FSGroupChangePolicy(value)
+				case "sensitiveMountOptions":
+					c.mounts[i].Options = append(c.mounts[i].Options, value)
```

Comment:
Also just thinking aloud here: we could remove this diff by collapsing `sensitiveMountOptions` into the regular options in the driver. I agree this is not a big deal, but just for my understanding, would there be any drawback to that (beside having to make that change in the driver)?

Reply:
In my opinion, it makes sense to keep the sensitive values separately as part of Metadata field of `MountInfo` struct. One scenario I can think of is when using sealed secrets for passing along these volume credentials, where this Metadata can be passed as `DriverOptions` field to CDH for handling unsealing secrets without having to walk the opts slice to retrieve the sensitive keys.